assets/src/edit-story/components/library/panes/media/media3p/api/apiFetcher.js

```diff
@@ -91,13 +86,16 @@ export async function listMedia({
   const queryString = new URLSearchParams(Object.fromEntries(new Map(params)));
   const url = API_DOMAIN + Paths.LIST_MEDIA + '?' + queryString.toString();
 
-  const data = await fetch(url);
+  const response = await window.fetch(url);
 
-  if (!data.length) {
-    throw new Error('Obtained an empty response from listMedia call.');
+  if (!response.ok) {
+    throw new Error(
+      'Obtained an error from the listMedia call, statusCode: ' +
+        response.status
```

Comment:
Shall we also log `response.statusText` ?

Reply:
Done.